boa/src/exec/tests.rs

```diff
@@ -486,11 +486,6 @@ fn unary_delete() {
         delete a
     "#;
     assert_eq!(&exec(delete_object), "false");
-
-    let delete_recursive = r#"
-        delete delete delete 1
-    "#;
-    assert_eq!(&exec(delete_object), "true");
```

Comment:
Hmmm why doesn't this work?

Reply:
The handler for `Node::Const` was missing